lib/app_web/controllers/sent_controller.ex

```diff
@@ -130,6 +130,15 @@ defmodule AppWeb.SentController do
   end
 
   def pixel(conn, _params) do
+    # warm up the lambda function so emails are sent instantly!
+    payload = %{"ping" => :os.system_time(:millisecond), key: "ping"}
+    # IO.inspect(payload, label: "payload ping/2:151")
+    # see: https://github.com/dwyl/elixir-invoke-lambda-example
+    lambda = System.get_env("AWS_LAMBDA_FUNCTION")
+    ExAws.Lambda.invoke(lambda, payload, "no_context")
+    |> ExAws.request(region: System.get_env("AWS_REGION"))
+    |> IO.inspect(label: "ExAws.Lambda ping response")
```

Comment:
Not sure if you want to keep this `IO.inspect` uncommented

Reply:
For now I'm happy to leave these kinds of logs in this App.
When we are _paying_ for the storage of logs, we will have to be strict about them
and will probably have a pre-commit hook/check to not allow uncommented `IO.inspect`. 👍